functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/BuildConfigurator.java

```diff
@@ -218,7 +226,7 @@ boolean isQuietMode() {
     }
 
     boolean isSkipTestsEnabled() {
-        return skipTests;
+        return skipTests || Boolean.valueOf(getPropertiesFromPom().getProperty("skipTests"));
```

Comment:
It is stupid but can you move this `Boolean.valueOf` sentence to a method called something like `isSkipTestsSetInPom` or something like this. Just to make it simple to read the conditional.